### DISCUSSION TITLE
Can't identify missing icon

### DISCUSSION THREAD

**Discussion by karambaq** [LazyVim/LazyVim#729] [Q&A]
When I'm typing `/` - it shows this in status line:

<img width="88" alt="image" src="https://user-images.githubusercontent.com/14233969/236846515-cd9d74ee-3536-4b94-ae36-8b690b68e125.png">

Where is this icon from?

**Comment by dpetka2001** [comment]
That icon should be a lense and it's being rendered by your patched font that you use. I don't know which font you use, but I use `SauceCode pro` from nerd-fonts. If you also use nerd-fonts, then you should probably update the font you use, because there were some big changes in latest updates and most plugins changed the icons they use to reflect those changes.

**Reply by karambaq** [reply]
All of the other icons are works fine, I patched my font a few days ago with Nerd font patcher:

<img width="1334" alt="image" src="https://user-images.githubusercontent.com/14233969/236853765-bc7a302f-1e98-4903-8c7b-15690b04f959.png">

**Reply by dpetka2001** [reply]
No other idea what it may be causing it. I'm not encountering this problem.

**Reply by karambaq** [reply]
@dpetka2001 Can you please send me this icon here? So I can find it in the repo and identify which plugin using it.

**Reply by dpetka2001** [reply]
Away from computer at the moment but if i recall correctly this is provided by noice.nvim plugin so take a look at the repo if it has any icons.

**Reply by folke** [reply]
You need an up to date nerd font.

**Reply by karambaq** [reply]
I don't have an option to update it, I patched it two days ago. Do you know which icon is this?

**Reply by karambaq** [reply]
Actually I just patched it again with new version of https://github.com/ryanoasis/nerd-fonts#font-patcher
Icon still not displaying.

**Reply by folke** [reply]
Then you didn't patch it correctly. Sorry. LazyVim and all my projects for that matter use the latest nerdfont symbols. I'm also not using deprecated symbols. You should check that your terminal is indeed using the nerd font symbol and that it's the latest version

**Reply by folke** [reply]
fyi, I use https://github.com/loichyan/nerdfix to always ensure my projects are up to date

**Reply by karambaq** [reply]
Strange. It works in default terminal but doesn't in iTerm2, thanks
